components/form/input/src/Input/Component/index.js

```diff
@@ -0,0 +1,72 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import './index.scss'
```

Comment:
I don't think this is allowed to do. scss should always be added manually. 
To me we can decide to use this way but I think it may imply problems with themes

Reply:
We should check alternatives, loading all styles from the main styles files makes us load css that might not be used